src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -1772,6 +1772,10 @@
   <data name="UnitAbbreviation_Joule" xml:space="preserve">
     <value>J</value>
     <comment>An abbreviation for a measurement unit of energy</comment>
+  </data>
+    <data name="UnitAbbreviation_Kilowatthour" xml:space="preserve">
+    <value>kWh</value>
+    <comment>An abbreviation for a measurement unit of energy</comment>
```

Comment:
nit: A more detailed description of the abbr. about ***Kilowatt-hour*** is much more appreciated.